src/neuroconv/tools/nwb_helpers/_metadata_and_file_helpers.py

```diff
@@ -10,9 +10,10 @@
 from typing import Literal, Optional
 from warnings import warn
 
+import h5py
```

Comment:
Remove unused imports.

Reply:
Yeah I started down this path initially, but found it was the wrong direction; you cleaned up in the other PR though